src/kirby/components/toolbar/toolbar.component.tns.html

```diff
@@ -0,0 +1,27 @@
+<ActionBar class="toolbar {{themeColor}}">
+  <NavigationButton  icon="~/assets/images/arrow-back.png" [visibility]="!hideBackButton && isAndroid ? 'visible' : 'collapsed'" (tap)="onBackButtonSelected()">
```

Comment:
As I am aware, We cannot, I am not sure wether it will change in v6.0, but for now I am pretty convinced the only way to have identical back arrows is by using a .png.